### ISSUE TITLE
Critical Error

### ISSUE THREAD

**Issue by Moreo18** [kihecido/BMBF#58]
(Sorry if I there is some mistakes, I'm a french guy)
I finally decided to update Beat Saber and BMBF and when I try to launch BMBF (after an installation without any trouble), It just loading for ever. And sometimes (or my pc) it say a thing like "critical error, try to reload assets in the tool section" but this section was unreachable on the quest view. I found it on my pc but when I was clicking on the button, a new windows appeared with a loading on it (endless of course). Is someone have an idea to solve my trouble ?
But, great job of the dev because in the old version, it work perfectly

**Comment by henwill8** [comment]
They havent updated BMBF yet for 1.8.0, I would suggest going back to 1.7.0 until they do.

**Comment by Moreo18** [comment]
Okay but how can i can go back to an older version of beat saber ?

**Comment by Rayn322** [comment]
Did you back up the 1.7 apk?

**Comment by Moreo18** [comment]
No because I never make the 1.7 update of BS because I didn't want to lose all my data. And I decided to make the upgrade, and here I am :'-(

**Comment by Moreo18** [comment]
So I've just read #59 and... I wait (I lose all my data to nothing :'-()

**Comment by BlazingFlame15** [comment]
Have you made sure that wifi is on in your quest? This happened to me and
it was as simple as that...

**Comment by Moreo18** [comment]
Yes because I can download beat saber from the store but it doesn't work

**Comment by Rayn322** [comment]
Make sure to always back up your save data no matter what before doing any modding. That's all of the scores and stuff if that's what you mean by "I lose all my data to nothing"

**Comment by Moreo18** [comment]
I made a backup but it was link to the old apk, unrestorable (I'd try)

**Comment by Moreo18** [comment]
If a nice people can re-up when the update will be release, that could be nice :-)

**Comment by Andreas-W-S** [comment]
i bought beat saber this week in the oculus store. i don't know which version of BS this is.
Currently i have installed BMBF V1.7.0. All i get is this critical error.
Does anyone already have an idea?

**Comment by Trae66** [comment]
Andreas what day did you buy it? Version 1.8 was released yesterday and if you want to keep your mods until BMBF gets updated, its advised not to update.

**Comment by Rayn322** [comment]
what are the newest songs in your game (check the main menu where you click solo or party, the bottom left has the newest songs)

**Comment by Moreo18** [comment]
BMBF is for 1.7 and now BS is in 1.8 version... Update don't work on modded BS.

**Comment by Andreas-W-S** [comment]
I bought it on 03/25/2020. The oculus app shows BS version 1.8.0. 

I had some bad luck i guess :-(

**Comment by Rayn322** [comment]
It shouldn’t be long. Plus, since you’ve just gotten the game I’m sure there’s plenty of songs that you haven’t played yet.

**Comment by Andreas-W-S** [comment]
All the songs that are included are not my taste at all. 
Seriously, I have trouble calling that music at all. I love the old style music. :-)

**Comment by Alfonsov11** [comment]
I have the same problem, I just bought beat saber and the oculus quest 2 days ago and the same luck happen to me with the BMBF on 1.7 and beat saber 1.8

**Comment by Moreo18** [comment]
Release 1.8.0 😀
